### PR TITLE
PAPP-25551: Adding new action 'create meeting'

### PR DESCRIPTION
**Pull request by dhwanis-crest**
Please ensure your pull request (PR) adheres to the following guidelines:

- Please refer to our contributing documentation for any questions on submitting a pull request, link: [Contribution Guide](https://github.com/splunk-soar-connectors/.github/blob/main/.github/CONTRIBUTING.md)

#### Please check if your PR fulfills the following requirements:
- [x] Testing of all the changes has been performed (for bug fixes / features)
- [x] The readme.html has been reviewed and added / updated if needed (for bug fixes / features)
- [x] Use the following format for the PR description: `<App Name>: <PR Type> - <PR Description>`
- [x] Provide release notes as part of the PR submission which describe high level points about the changes for the upcoming GA release.
- [x] Verify all checks are passing.
- [x] Do NOT use the `next` branch of the forked repo. Create separate feature branch for raising the PR.
- [x] Do NOT submit updates to dependencies unless it fixes an issue.
#### Please check the type of change your PR introduces:
- [ ] New App
- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation
- [ ] Other (please describe): 

## Security Considerations (REQUIRED)
- [ ] If you are exposing any endpoints using a [REST handler](https://docs.splunk.com/Documentation/SOAR/current/DevelopApps/RESTHandlers), 
  please document them in the `readme.html`.
- [ ] If this is a new connector or you are adding new actions
    - [ ] Please document in the `readme.html` all methods (eg, OAuth) used to authenticate 
      with the service that the connector is integrating with.
    - [ ] If any actions are unable to run on SOAR Cloud, please document this in the `readme.html`.
- [ ] Are you introducing any new cryptography modules? If yes, please elaborate their purpose:
- [ ] Are you are accessing the file system? If yes, please verify that you are only accessing paths returned through 
the [Vault](https://docs.splunk.com/Documentation/SOARonprem/5.2.1/DevelopApps/AppDevAPIRef#Vault) API.
- [ ] Are you are marking code to be ignored by Semgrep with [`nosemgrep`](https://semgrep.dev/docs/ignoring-files-folders-code/#ignoring-code-through-nosemgrep)?
If yes, please provide justification in an additional comment next to the ignored code.

  

## Release Notes (REQUIRED)
- Provide release notes as part of the PR submission which describe high level points about the changes for the upcoming GA release.

## What is the current behavior? (OPTIONAL)
- Describe the current behavior that you are modifying.

## What is the new behavior? (OPTIONAL)
- Describe the behavior or changes that are being added by this PR.

## Other information (OPTIONAL)
- Any other information that is important to this PR such as screenshots of how the component looks before and after the change.

## Pay close attention to (OPTIONAL)
- Any specific code change or test case points which must be addressed/reviewed at the time of GA release.

## Screenshots (if relevant)

---
Thanks for contributing!

### CODE REVIEW

File: microsoftteams_connector.py [hunk 1]
```diff
@@ -253,6 +254,11 @@ def _handle_login_response(request):
 
     # If value of admin_consent is not available, value of code is available
     state['code'] = code
+    try:
+        state['code'] = MicrosoftTeamConnector().encrypt_state(code, "code")
+        state[MSTEAMS_STATE_IS_ENCRYPTED] = True
+    except Exception as e:
+        return HttpResponse("{}: {}".format(MSTEAMS_DECRYPTION_ERR, str(e)), content_type="text/plain", status=400)
```
Comment (jdemelo) [inline microsoftteams_connector.py:261]: Should this be `MSTEAMS_ENCRYPTION_ERR` to correspond to `encrypt_state()` call?

File: microsoftteams_connector.py [hunk 2]
```diff
@@ -642,14 +685,19 @@ def _generate_new_access_token(self, action_result, data):
...
-        if (self._access_token != self._state.get("token", {}).get(MSTEAMS_ACCESS_TOKEN_STRING)) or (self._refresh_token != self._state.get
-        ("token", {}).get(MSTEAMS_REFRESH_TOKEN_STRING)):
-            message = "Error occurred while saving the newly generated access or "
-            message += "refresh token (in place of the expired token) in the state file."
-            message += " Please check the owner, owner group, and the permissions of the state file. The Phantom "
-            message += "user should have the correct access rights and "
-            message += "ownership for the corresponding state file (refer to readme file for more information)."
-            return action_result.set_status(phantom.APP_ERROR, message)
+        try:
+            if self._access_token != self.decrypt_state(self._state.get("token", {}).get
```
Comment (jdemelo) [inline microsoftteams_connector.py:689]: I think there are a couple `"token"` literals here that could be `MSTEAMS_TOKEN_STRING`

Comment (dhwanis-crest) [reply microsoftteams_connector.py]: Updated

File: microsoftteams_connector.py (hunk 1 above)
Comment (dhwanis-crest) [reply microsoftteams_connector.py]: Updated

**Review by ricardo-phantom — APPROVED** [review]

**Review by alexa-phantom — APPROVED** [review]

**Pull request merged**